pandatool/src/deploy-stub/deploy-stub.c

```diff
@@ -628,6 +626,62 @@ static void unmap_blob(void *blob) {
   }
 }
 
+/**
+ * Gets the file path to the games .bin file which stores the blob made by FreezeTool
+ */
+static void get_blob_path(const char *path, char buffer[]) {
```

Comment:
This seems to duplicate a fair bit of logic from `map_blob() `. Maybe create a `get_runtime_path()` to put some of that duplicate code into?